code/__DEFINES/flags.dm

```diff
@@ -91,3 +91,24 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 // radiation
 #define RAD_PROTECT_CONTENTS (1<<0)
 #define RAD_NO_CONTAMINATE (1<<1)
+
+//Mob mobility var flags
+/// can move
+#define MOBILITY_MOVE			(1<<0)

```

Comment:
Shouldn't the descriptions of the flags be in the same line for better reading and to know more easily what do what?

Reply:
Tg code documentation system